src/platform/android/BUILD.gn

```diff
@@ -19,21 +19,24 @@ import("${chip_root}/src/platform/device.gni")
 
 import("${build_root}/config/android_abi.gni")
 import("${chip_root}/build/chip/java/rules.gni")
+import("${chip_root}/src/tracing/tracing_args.gni")
 
 assert(chip_device_platform == "android")
 
 source_set("tracing") {
-  sources = [
-    "tracing.cpp",
-    "tracing.h",
-  ]
-
-  deps = [
-    "${chip_root}/src/lib/support",
-    "${chip_root}/src/tracing",
-    "${chip_root}/src/tracing/perfetto:event_storage",
-    "${chip_root}/src/tracing/perfetto:simple_initialization",
-  ]
+  deps = [ "${chip_root}/src/tracing" ]
+
+  if (matter_enable_tracing_support) {
+    sources = [
+      "tracing.cpp",
+      "tracing.h",
+    ]
+
+    deps += [
```

Comment:
lib/support/logging/CHIPLogging.h is included by the header. It seems the dependency on src/lib/support should be kept

Reply:
> It seems the dependency on src/lib/support should be kept

src/lib/support is provided by `${chip_root}/src/tracing` public_deps